modules/exploits/hpux/lpd/cleanup_exec.rb

```diff
@@ -3,95 +3,96 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
+require 'English'
 class MetasploitModule < Msf::Exploit::Remote
   Rank = ExcellentRanking
 
   include Msf::Exploit::Remote::Tcp
 
   def initialize(info = {})
-    super(update_info(info,
-      'Name'           => 'HP-UX LPD Command Execution',
-      'Description'    => %q{
+    super(
+      update_info(
+        info,
+        'Name' => 'HP-UX LPD Command Execution',
+        'Description' => %q{
           This exploit abuses an unpublished vulnerability in the
-        HP-UX LPD service. This flaw allows an unauthenticated
-        attacker to execute arbitrary commands with the privileges
-        of the root user. The LPD service is only exploitable when
-        the address of the attacking system can be resolved by the
-        target. This vulnerability was silently patched with the
-        buffer overflow flaws addressed in HP Security Bulletin
-        HPSBUX0208-213.
-      },
-      'Author'         => [ 'hdm' ],
-      'References'     =>
-        [
-          [ 'CVE', '2002-1473'],
-          [ 'OSVDB', '9638'],
-          [ 'URL', 'http://archives.neohapsis.com/archives/hp/2002-q3/0064.html'],
-
+          HP-UX LPD service. This flaw allows an unauthenticated
+          attacker to execute arbitrary commands with the privileges
+          of the root user. The LPD service is only exploitable when
+          the address of the attacking system can be resolved by the
+          target. This vulnerability was silently patched with the
+          buffer overflow flaws addressed in HP Security Bulletin
+          HPSBUX0208-213.
+        },
+        'Author' => [ 'hdm' ],
+        'References' => [
+          ['CVE', '2002-1473'],
+          ['OSVDB', '9638'],
+          ['URL', 'https://web.archive.org/web/20041213153521/http://archives.neohapsis.com/archives/hp/2002-q3/0064.html'],
```

Comment:
cc https://github.com/rapid7/metasploit-framework/pull/19896 - the script likely needs updated to handle whatever nuance is happening here, as it should've caught this link